department-of-veterans-affairs/vets-website
# MHV-61933 Download-The-Self-Entered-Records-PDF-Or-TXT

## Summary
Added scaffolding to enable downloading of self-entered records in PDF or TXT format.

## Related issue(s)

[MHV-61933](https://jira.devops.va.gov/browse/MHV-61933) - Download the self entered records (PDF or TXT)

Download the self entered records (PDF or TXT)

Data sources are in ticket [MHV-64049](https://jira.devops.va.gov/browse/MHV-64049) Download the self entered records-data sources for the PDF or TXT page - VA DTE Jira

User Story: As a Medical Records User, I want to Download the self entered records

GIVEN

WHEN

THEN

Feature Flag Y/N ?

DataDog/Analytics Y/N ?

Manual Testing Y/N ? 

Automated Testing Y/N ?

Accessibility Testing Y/N ?

UCD Validation Y/N

 

Acceptance Criteria:

AC1  Download functionality has been created for Self entered Health History (Data sources in separate ticket)

AC2

AC3
AC4
AC5

Links to UCD:

Figma Link:

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

## Testing done
None

## Screenshots
None


## What areas of the site does it impact?
None

## Acceptance criteria
Scaffolding for the PDF or TXT page .

### Quality Assurance & Testing


### Authentication


## Summary
Update lkfja)

[MHV-61246](https://jira.devops.va.gov/browse/MHV-61246) - Add the "Last updated" text to the print view for List pages

Add the "Last updated" text to the print view for List pages

The NewRecordsIndicator has been removed from the print view, because it was printing the spinner, but we probably want to print something there.

Work with UCD to decide how it should look when printed in various scenarios, then implement inside NewRecordsIndicator.

User Story:  As a MR user, I want to    so that I can


GIVEN:

WHEN:

THEN:


Feature Flag Y/N ?

DataDog Analytics  Y/N ?

Manual Testing Y/N ? 

Automated Testing Y/N ?

Accessibility Testing Y/N ?

UCD Validation Y/N

Acceptance Criteria:

AC1

AC2

AC3

AC4

Links to UCD:

Figma Link:

Other Design Notes:
## Testing done
None

## Screenshots

## What areas of the site does it impact?
The modification affects the Vitals Detail page in Medical Records pages.

## Acceptance criteria
The focus shifts back to the title when pagination navigates to the first page. 

### Quality Assurance & Testing


### Authentication
